File: app/src/main/java/com/example/mitroshin/pagingdemo/model/entity/Links.java
package com.example.mitroshin.pagingdemo.model.entity;

import com.google.gson.annotations.SerializedName;

public class Links {

    @SerializedName("self")
    public final String self;

    @SerializedName("html")
    public final String html;

    @SerializedName("download")
    public final String download;

    @SerializedName("download_location")
    public final String downloadLocation;

    public Links(String self,
                 String html,
                 String download,
                 String downloadLocation) {
        this.self = self;
        this.html = html;
        this.download = download;
        this.downloadLocation = downloadLocation;
    }

    @Override
    public String toString() {
        return "Links{" +
                "self='" + self + '\'' +
                ", html='" + html + '\'' +
                ", download='" + download + '\'' +
                ", downloadLocation='" + downloadLocation + '\'' +
                '}';
    }
}

File: app/src/main/java/com/example/mitroshin/pagingdemo/model/entity/Links.kt
package com.example.mitroshin.pagingdemo.model.entity

import com.google.gson.annotations.SerializedName

class Links(
        @field:SerializedName("self")
        val self: String,

        @field:SerializedName("html")
        val html: String,

        @field:SerializedName("download")
        val download: String,

        @field:SerializedName("download_location")
        val downloadLocation: String)